test/unit/path.js

```diff
@@ -48,12 +48,16 @@
     return el;
   }
 
-  function getPathObject(path, callback) {
-    fabric.Path.fromElement(getPathElement(path), callback);
-  }
-
   function makePathObject(callback) {
-    getPathObject('M 100 100 L 300 100 L 200 300 z', callback);
+    const path = new fabric.Path('M 100 100 L 300 100 L 200 300 z', {
+      fill: 'red',
+      stroke: 'blue',
+      strokeLineCap: 'butt',
+      strokeLineJoin: 'miter',
+      strokeMiterLimit: 4,
+      strokeWidth: 0,
+    });
+    callback(path);
```

Comment:
no reason for this to be a cb

Reply:
i think the issue was changing too many tests